HW0/P4/P4.py

```diff
@@ -29,20 +29,27 @@
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
+    s_true = np.loadtxt('P4_trajectory.txt', delimiter=',')
     #####################
 
     # ax.plot(x_coords, y_coords, z_coords,
     #         '--b', label='True trajectory')
-
+    ax.plot(s_true[:,0], s_true[:,1], s_true[:,2], '--b', label='True trajectory')
+    
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
+    s_obs = np.loadtxt('P4_measurements.txt', delimiter=',')
+    R = np.array([[1.0/rx, 0, 0], [0,1.0/ry,0], [0,0,1.0/5.0]])
```

Comment:
This is a small detail, but I would reference rz (as you did with rx and ry) instead of hard-coding it.